modules/mod_ginger_rdf/support/rdf_property.erl

```diff
@@ -4,20 +4,44 @@
 -export([
     'dbpedia-owl'/1,
     foaf/1,
-    rdfs/1
+    geo/1,
+    rdf/1,
+    rdfs/1,
+    schema/1,
+    vcard/1
 ]).
 
 -include_lib("zotonic.hrl").
 -include_lib("../include/rdf.hrl").
 
+-spec 'dbpedia-owl'(binary()) -> binary().
```

Comment:
Is it possible to be more "expressive" here (and below) by using a type that is an alias for `binary()`?